Examples/05_openfast_sim.py

```diff
@@ -10,7 +10,7 @@
 
 Note
 
-* you will need to have a compiled controller in ROSCO/build/ 
+* you will need to have a compiled controller in ROSCO/rosco/controller/build
```

Comment:
Would it be better to point to the default location of rosco library : 'ROSCO/rosco/lib/'?